packages/flutter/lib/src/foundation/change_notifier.dart

```diff
@@ -264,6 +262,7 @@ class ChangeNotifier implements Listenable {
       _debugDisposed = true;
       return true;
     }());
+    _listeners = const <VoidCallback?>[];
```

Comment:
Let's follow the performance-optimized pattern suggested in https://github.com/flutter/flutter/pull/71947/files#r545722476 here.

Reply:
sounds good, I will also leave a comment as the original pr suggested..